wpeverest/user-registration
# Ur 1054 Enhancement - Custom password and Realtime password strength guide

### All Submissions:






### Changes proposed in this Pull Request:


Added a new custom password strength option.
Closes # .

### How to test the changes in this Pull Request:

1. Form Settings -> General -> Enable Strong Password
2. Select Custom Option -> Fill fields -> check if the validation works.

### Types of changes:




### Other information:




### Changelog entry

> Custom password and Realtime password strength guide.
